### PR TITLE
Some (very) little improvement for the UI

### PR DESCRIPTION
**Pull request by nhandt64**
I'm a bit bored so... yeah, that's how this PR come up

Preview:

image

Notice:
1. The blur background may break sometimes with animations on Chrome. 
2. Firefox still doesn't support the blur yet (but will in near future).
3. I don't know where did I got this wallpaper, pretty sure the current state is public domain.
4. This PR is made in my free time, I don't think I'm gonna work for this project in the (near) future.

HF!

### CODE REVIEW

**Comment by ghost** [comment]
Woah this looks fantastic! I'll stick this to Edaz to decide though.

**Comment by nhandt64** [comment]
Please also note that I didn't spend any effort on doing this. So yeah, it may look a bit suck, somehow.
_The improvenment that could be suck._

**Comment by edazpotato** [comment]
First off, It looks amazing! Thank you for taking the time to contribute to one of our projects!

However, there are a couple of issues.
1. Some people will white text on a light grey background difficult to read
2. Image rights. I love the picture, and it looks super cool, but sadly we can't use it unless we are certain it's public domain or the artist has granted us special permission (there are some massive fines we could face if it isn't public domain or used with permission).

Basically, If you can improve the text legibility find the source of the image location to ensure that it definitely is public domain I'll merge it.

**Comment by nhandt64** [comment]
About the text: yes, I've changed it between white and black for many times before save up and make the commit. I mainly focus on the div background.
About the background: it's up to you. I've used many tools to hunt its origin and no luck so far. IIRC, I took it from reddit.
Edit: I will find the place where I took the background. If I found nothing (in about 24 hours), feel free to remove the background or I'll use another one that I'm sure it's free to use in the next commit.

**Comment by nhandt64** [comment]
The last background is pretty confused, it - I believe that - became a public domain image for a long time that a random Russian took and sell it as his own stuff on Adobe Stock.

Annnnnndddd I have found out that I forgot to turn of the dark theme plugin (automatically modify website CSS to dark theme) so the last preview image looks pretty suck.

Here's the preview of this commit:
image
Isn't it too bright?

Source of image: https://pixabay.com/photos/bubbles-water-bubbly-bubbling-230014/
Pixabay is a free service that provides free images for all purposes without asking for permission/credit the author/etc.

**Comment by niilasoika** [comment]
Hey! Thanks for your support!

I see you have been discussing with our other staff. I will still stick to Edaz to review this, but I approve this.
I don't personally think it's too bright, in my opinion it's good.

As I stated earlier, I am waiting for Edaz. Thank you a lot!

- Niilas

**Review by edazpotato — APPROVED** [review]
Thank you again for taking the time to contribute.
The Image looks great. Thanks for making sure you got a public domain one 👍.

I was still worried about the text legibility though, especially with the brighter image so I increased the main section's background opacity a little.

**Comment by nhandt64** [comment]
Glad you liked it with your little modify 👍.
I'm also a bit interested in your guys development. Let me know if your team does have a place to talk and work like a Discord server and welcome me as well. So that I can give a hand and/or some tips in my free time.

Anyways, merge if you feel happy with it!

**Comment by edazpotato** [comment]
If you want to hang out with us, we do have a discord server! https://discord.gg/ZSh8ZpTM6a
